packer/deps.bzl

```diff
@@ -22,14 +22,12 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 def deps():
     http_archive(
         name = "packer_osx",
-        url = "https://releases.hashicorp.com/packer/1.7.4/packer_1.7.4_darwin_amd64.zip",
-        sha256 = "f3faf9dce0cebdfc7abfcf70511c6230e0c0a5c499ca3478def81549ded91b20",
-        build_file_content = 'exports_files(["packer"])'
+        build_file_content = 'exports_files(["packer"])',
```

Comment:
Is it possible to keep the `sha256`field and update the value, rather than removing them completely?

Reply:
Done!